src/app/constants/index.ts

```diff
@@ -0,0 +1,3 @@
+export const EMAIL_REGEX = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
```

Comment:
- The current email regex is simpler but misses some valid cases, like quoted emails ("example"@domain.com) and domain literals (user@[192.168.1.1]). The previous stricter regex in `user.invitation.component.ts` ensured better compliance with email standards, reducing false rejections. kindly consider keeping the previous instead of switching to a looser pattern. and changing this was out scope of what the ticket says